ocs_ci/templates/ocs-deployment/ceph-debug-log-level-configmap.yaml

```diff
@@ -0,0 +1,19 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rook-config-override
+  namespace: openshift-storage
+data:
+  config: |
+    [global]
+    mon_osd_full_ratio = .85
```

Comment:
Are `mon_osd_full_ratio`, `mon_osd_backfillfull_ratio`, `mon_osd_nearfull_ratio`, `mon_max_pg_per_osd` and `osd_memory_target_cgroup_limit_ratio` needed to be set in `rook-config-override` as part of debug configuration?

Reply:
I think the entire configuration is needed, and we can't change a single value at a time 
@travisn right?

Reply:
This is not a theoretical problem, for example we changed full ratios in the past.

Reply:
Yes, it would be best if you can fetch the default config from the OCS operator and then append to it. This way, if OCS adds other settings to the default config it wouldn't be missed in the tests.

Reply:
How can we fetch the Ceph configuration dynamicallyy?

Reply:
I dont think the default config is changed when debug settings are applied ? @travisn why would default config change with this setting?

Reply:
@vasukulkarni we are not changing the default config apart from setting the log level to debug. All the rest of the configuration that exists in this configMap is needed because AFAIU, for changing something in the Ceph config, you need to pass all the rest of the configuration parameters, which are not the default in Ceph and are modifications done for OCS

Reply:
So if none of the other configurations are changed then there really isn't an issue, one could do config dump (normal vs debug) to check as well.

Reply:
Ceph config will be validated by comparing it between the rook config override configmap and what we have stored in constants.
Resolving

Reply:
marking as resolved as this is now addressed in the new test case